lib/src/base58/base58.dart

```diff
@@ -308,7 +308,7 @@ List<int> decode(String value) {
     int carry = _reverseMap[currentByte];
     int i = 0;
     if (carry == -1) {
-      throw ArgumentError('invalid base58 character found $currentByte');
+      throw Exception('invalid base58 character found `$currentByte\'');
```

Comment:
`FormatException` would be better here, this is what e.g. `int.parse()` throws for malformed input.